main.tf

```diff
@@ -106,6 +106,39 @@ resource "aws_lb_target_group" "https" {
   }
 }
 
+# secondary target used with code deploy
+resource "aws_lb_target_group" "https2" {
```

Comment:
Along the lines of wha Chas is saying. Could you just set a count on the original `aws_lb_target_group` resource and have the count default to 1?

Reply:
I made some changes locally to test the count method and it does work. Answering @cblkwell's question it will most definitely have a breaking change as opposed to what this PR currently is doing.

The method of doing so was to remove the target groups from the ALB and let TF tear the existing target group and built up from scratch. Thoughts?